bot/constants.py

```diff
@@ -47,6 +47,7 @@ class EnvConfig(
 
 
 class _Channels(EnvConfig, env_prefix="channels_"):
+    general: int = 123123123123
```

Comment:
Anything changes made just for testing should be reverted (though I'm not really sure why this would be needed, when you can just use a channel that is already configured as whitelisted)